### PR TITLE
Update Testerina BBEs

### PR DESCRIPTION
**Pull request by Ibaqu**
## Purpose
Update Testerina BBEs

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: examples/testerina-mocking-objects/testerina_mocking_objects_test.out [hunk 1]
```diff
@@ -5,9 +5,6 @@ ballerina test bbe_mocking
 Compiling source
         ballerinatest/bbe_mocking:0.1.0
 
-Creating balos
```
Comment (Dilhasha) [inline examples/testerina-mocking-objects/testerina_mocking_objects_test.out:8]: @Ibaqu if you are considering this to a single file test execution, we might need to update the package details under "compiling source" section. Can you verify this?

Comment (Ibaqu) [reply examples/testerina-mocking-objects/testerina_mocking_objects_test.out:8]: Verified with the latest pack. `Creating balos` has been removed.

**Review by Dilhasha — APPROVED** [review]

File: examples/testerina-mocking-objects/testerina_mocking_objects_main.bal [hunk 2]
```diff
@@ -29,13 +29,13 @@ function performGet() returns @tainted http:Response {
 // This function sends out email to specified email addresses
```
Comment (praneesha) [inline examples/testerina-mocking-objects/testerina_mocking_objects_main.bal:29]:
```suggestion
// This function sends out an email to the specified email addresses
```

File: examples/testerina-mocking-objects/testerina_mocking_objects_test.bal [hunk 3]
```diff
@@ -45,7 +45,7 @@ function testReturnWithArgs() {
     test:prepare(clientEndpoint).when("get").
     withArguments("/headers").thenReturn(mockResponse);
     // The object and record types should be denoted by the `test:ANY` constant
```
Comment (praneesha) [inline examples/testerina-mocking-objects/testerina_mocking_objects_test.bal:47]:
```suggestion
    // The object and record types should be denoted by the `test:ANY` constant.
```

File: examples/testerina-mocking-objects/testerina_mocking_objects_test.bal [hunk 4]
```diff
@@ -71,7 +71,7 @@ function testSendNotification() {
     smtpClient = test:mock(email:SmtpClient);
     // This stubs the `send` method of the `mockSmtpClient` to do nothing.
     // This is used for functions with an optional or no return type
```
Comment (praneesha) [inline examples/testerina-mocking-objects/testerina_mocking_objects_test.bal:73]:
```suggestion
    // This is used for functions with an optional or no return type.
```

File: examples/testerina-mocking-objects/testerina_mocking_objects_main.bal (hunk 2 above)
Comment (Ibaqu) [reply examples/testerina-mocking-objects/testerina_mocking_objects_main.bal]: Addressed comment

File: examples/testerina-mocking-objects/testerina_mocking_objects_test.bal (hunk 3 above)
Comment (Ibaqu) [reply examples/testerina-mocking-objects/testerina_mocking_objects_test.bal]: Address comment

File: examples/testerina-mocking-objects/testerina_mocking_objects_test.bal (hunk 4 above)
Comment (Ibaqu) [reply examples/testerina-mocking-objects/testerina_mocking_objects_test.bal]: Addressed comment

**Review by praneesha — APPROVED** [review]

**Pull request merged**